tools/travis/run_codespell.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+sudo pip install codespell
```

Comment:
Ah, I forgot one thing, might want to add `set -e` so we don't have hidden errors.